app/pb/communities.proto

```diff
@@ -64,6 +64,22 @@ service Communities {
     //
     // Raises: FAILED_PRECONDITION if the user is not part of the community or if their location is within the community
   }
+
+  rpc ListUserCommunities(ListUserCommunitiesReq) returns (ListUserCommunitiesRes) {
+    // Gets a list of communities a member belongs to
+  }
+
+  rpc ListUserGroups(ListUserGroupsReq) returns (ListUserGroupsRes) {
+    // Gets a list of groups a member belongs to
+  }
+
+  rpc ListUserPlaces(ListUserPlacesReq) returns (ListUserPlacesRes) {
+    // Gets a list of groups a member belongs to
+  }
+
+  rpc ListUserGuides(ListUserGuidesReq) returns (ListUserGuidesRes) {
+    // Gets a list of groups a member belongs to
+  }
```

Comment:
Please move groups to `groups.proto`, and places/guides to `pages.proto`

Reply:
should I also move the methods to services/groups.py and services/pages.py?

Reply:
Yeah, if you move the RPC def to those protos, you have to move servicers as well